schemas/Quote.json

```diff
@@ -4,6 +4,10 @@
   "description": "A quote object",
   "type": "object",
   "properties": {
+    "source_connector_account": {
+      "description": "the connector's account on the source ledger",
+      "$ref": "Iri.json"
+    },
     "source_ledger": {
```

Comment:
No, I suppose not. I'll remove the ledgers. I think leaving both amounts in is simpler, though.